tests/e2e/lib.sh

```diff
@@ -368,21 +368,17 @@ setup_generated_certs_for_test() {
 
 setup_podsecuritypolicies_config() {
     info "Set POD_SECURITY_POLICIES variable based on kubernetes version"
-    local version
-    version=$(kubectl version --output json)
-    local majorVersion
-    majorVersion=$(echo "$version" | jq -r .serverVersion.major)
-    local minorVersion
-    minorVersion=$(echo "$version" | jq -r .serverVersion.minor)
-
-    # PodSecurityPolicy was removed in version 1.25
-    if (( "$majorVersion" >= 1 && "$minorVersion" >= 25 )); then
-        ci_export "POD_SECURITY_POLICIES" "false"
-        info "POD_SECURITY_POLICIES set to false"
-    else
+    available_api_resources=$(kubectl api-resources -o name)
+
+    if echo "$available_api_resources" | grep -q podsecuritypolicies.policy;
```

Comment:
1. I find a few examples of a slightly different pattern

https://github.com/stackrox/stackrox/blob/8f0e80513da43123d1e87bdef61ca13e4b8a1d5b/deploy/common/k8sbased.sh#L198-L202

Would be good to standardize on some approach and use it consistently. Please choose either your or other approach and make sure all instances look and work similarly.

2. [Nitpicking] There's no point to declare `available_api_resources` and abandon it later. It can be just inlined:

```bash
    if kubectl api-resources -o name | grep -q podsecuritypolicies.policy;
```

Reply:
Thanks for the hint, I changed my code to use the existing pattern.